e2e-robot/tests/microshift.robot

```diff
@@ -0,0 +1,138 @@
+
+Load Balancer Smoke Test
+    [Documentation]    Verify that Load Balancer correctly exposes HTTP service
+    [Tags]    smoke
+    [Setup]    Run Keywords    Create Hello MicroShift Pod    Expose Hello MicroShift Pod Via LB
+    Wait Until Keyword Succeeds    3x    3s    Access Hello Microshift via LB
+    [Teardown]    Delete Hello MicroShift Pod Route And Service
+
+Reboot Test
+    [Documentation]    Verify that MicroShift starts successfully after reboot
+    [Setup]    Run Keywords    Create Pod With PVC
+
+    Open Connection    ${USHIFT_IP}
+    Login    ${USHIFT_USER}    allow_agent=True
+    Execute Command    reboot now    sudo=True
+    Close Connection
+
+    Sleep    15s
+
+    Open Connection    ${USHIFT_IP}
+    Set Client Configuration    timeout=600s
+    Wait Until Keyword Succeeds    10x    10s    Login    ${USHIFT_USER}    allow_agent=True
+
+    Wait Until Keyword Succeeds
+    ...    10x
+    ...    10s
+    ...    Execute Command
+    ...    [ $(systemctl show -p SubState --value microshift) = "running" ]
+    ...    timeout=10s    return_stdout=True    return_stderr=True
+
+    ${stdout}    ${rc}=    Execute Command
+    ...    /etc/greenboot/check/required.d/40_microshift_running_check.sh | tee /tmp/asd.log
+    ...    sudo=True
+    ...    timeout=600s
+    ...    return_stdout=True
+    ...    return_rc=True
+    Log    ${stdout}
+    Should Be Equal As Integers    ${rc}    0
+    Close Connection
+
+    Run With Kubeconfig    oc wait --for=condition=Ready --timeout=120s pod/test-pod
+
+    [Teardown]    Delete Pod With PVC
+
+
+
+*** Keywords ***
+Get Kubeconfig
+    [Documentation]    X
+    Open Connection    ${USHIFT_IP}
+    Login    ${USHIFT_USER}    allow_agent=True
+    ${konfig}=    Execute Command
+    ...    cat /var/lib/microshift/resources/kubeadmin/${USHIFT_IP}/kubeconfig
+    ...    sudo=True
+    Should Not Be Empty    ${konfig}
+    ${rand}=    Generate Random String
+    ${path}=    Join Path    /tmp    ${rand}
+    Create File    ${path}    ${konfig}
+    Close Connection
+    Set Suite Variable    \${KUBECONFIG}    ${path}
+
+Remove Kubeconfig
+    Remove File    ${KUBECONFIG}
+
+Access Hello Microshift via Router
+    ${result}=    Run Process
+    ...    curl -i http://hello-microshift.cluster.local --resolve "hello-microshift.cluster.local:80:${USHIFT_IP}"
+    ...    shell=True    timeout=15s
+    Check HTTP Response    ${result}
+
+Access Hello Microshift via LB
+    ${result}=    Run Process    curl -i ${USHIFT_IP}:5678    shell=True    timeout=15s
+    Check HTTP Response    ${result}
+
+Check HTTP Response
+    [Arguments]    ${result}
+    Log    ${result.stdout}
+    Log    ${result.stderr}
+    Should Be Equal As Integers    ${result.rc}    0
+    Should Match Regexp    ${result.stdout}    HTTP.*200
+    Should Match    ${result.stdout}    *Hello MicroShift*
+
+Create Pod With PVC
+    Run With Kubeconfig    oc create -f ../e2e/tests/assets/pod-with-pvc.yaml
```

Comment:
Are these paths relative to the directory where someone runs robot, or to the location of the `.robot` file?

Reply:
I think it's relative to current working dir which, when running `make` inside `e2e-robot`, is `e2e-robot/`